cli/cli.ts

```diff
@@ -3853,7 +3857,7 @@ function stringifyTranslations(strings: pxt.Map<string>): string {
 }
 
 export function staticpkgAsync(parsed: commandParser.ParsedCommand) {
-    const route = parsed.flags["route"] as string || ".";
+    const route = parsed.flags["route"] as string || "/";
```

Comment:
@mmoskal @pelikhan:
According to https://makecode.com/cli/staticpkg, `route`, if missing, defaults to `local`. My understanding of `local` is that it refers to the "current directory", which means that if the output files are put on https://www.example.com/makecode/, all relative paths will be based on https://www.example.com/makecode/. If the value is changed to `/`, the base URL will be changed to https://www.example.com/. Is that intended?

Reply:
Um... Perhaps my understanding was incorrect, probably misled by the previous value `.`. The change to `/` makes the behavior consistent with that described in "Deploying PXT with static files".

I think both `./` in Line 3879 and the `local` on https://makecode.com/cli/staticpkg should be changed to `/`.

Have a question though: Should the use `route` be deprecated in favor of the generation of paths relative to the "current directory"? The advantage is that, when the routing is changed in the future, no re-compilation (or packaging) will be required.